ru/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -1,53 +1,53 @@
-<!-- Entities for AccountWizard -->
+﻿<!-- Entities for AccountWizard -->
 
-<!ENTITY windowTitle.label "Мастер учётных записей">
+<!ENTITY windowTitle.label "Мастер учетных записей">
 <!ENTITY accountWizard.size "width: 50em; height: 40em;">
 <!ENTITY macAccountWizard.size "width: 42em; height: 40em;">
 
 <!-- Entities for Account Type page -->
 
-<!ENTITY accountSetupInfo.label "Чтобы получать сообщения, сначала нужно создать учётную запись почты или новостей.">
-<!ENTITY accountTypeTitle.label "Параметры новой учётной записи">
-<!ENTITY accountTypeDesc.label "Мастер учётных записей соберёт информацию, необходимую для создания учётной записи почты или новостей. Если вы не знаете всей необходимой информации, свяжитесь с вашим системным администратором или поставщиком услуг Интернета.">
-<!ENTITY accountTypeDirections.label "Выберите, какой тип учётной записи вы хотите создать:">
-<!ENTITY accountTypeMail.label "Учётная запись электронной почты">
+<!ENTITY accountSetupInfo.label "Для получения сообщений сначала необходимо настроить учетную запись почты или группы новостей.">
+<!ENTITY accountTypeTitle.label "Настройка новой учетной записи">
+<!ENTITY accountTypeDesc.label "Этот мастер соберет информацию, необходимую для настройки учетной записи почты или группы новостей. Если необходимая информация неизвестна, обратитесь к системному администратору или поставщику услуг Интернета.">
+<!ENTITY accountTypeDirections.label "Выберите тип учетной записи, которую необходимо настроить:">
+<!ENTITY accountTypeMail.label "Учетная запись электронной почты">
 <!ENTITY accountTypeMail.accesskey "m">
-<!ENTITY accountTypeNews.label "Учётная запись новостей">
+<!ENTITY accountTypeNews.label "Учетная запись группы новостей">
 <!ENTITY accountTypeNews.accesskey "w">
```

Comment:
CORRECTIONS NEEDED for multiple lines with "учётная запись": if we agree on using "ё", let's use it everywhere.
Could you, please, also check all these green lines for double spaces used instead of single spaces?